net-dialup/freeradius/freeradius-3.0.19.ebuild

```diff
@@ -0,0 +1,244 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
+inherit autotools pam python-single-r1 systemd user
+
+MY_P="${PN}-server-${PV}"
+
+DESCRIPTION="Highly configurable free RADIUS server"
+SRC_URI="
+	ftp://ftp.freeradius.org/pub/radius/${MY_P}.tar.gz
+	ftp://ftp.freeradius.org/pub/radius/old/${MY_P}.tar.gz
+"
+HOMEPAGE="http://www.freeradius.org/"
+
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~sparc ~x86"
+LICENSE="GPL-2"
```

Comment:
Please check license. Source code indicates ```GPL-2+```.

Reply:
@ConiKost thanks.
So, GPL2 v2 = GPL-2+?

Reply:
Nope :-) The problem is, there are two different types. Some projects says, only GPL+2, others instead says GPL+2 or newer. Hence, we have two values for LICENSE.

Reply:
I follow what is wrote [here](https://github.com/gentoo/gentoo/blob/master/licenses/GPL-2%2B). Fixed in a separate commit for all releases. I saw that all ebuild versions are with GPL2 v2.

Let me know if it's needed a -r* if we change LICENSE field.

Thanks.